lib/src/facts/bsd/filesystem_resolver.cc

```diff
@@ -37,8 +37,10 @@ namespace facter { namespace facts { namespace bsd {
             point.name = fs.f_mntonname;
             point.device = fs.f_mntfromname;
             point.filesystem = fs.f_fstypename;
-            point.size = fs.f_bsize * fs.f_blocks;
-            point.available = fs.f_bsize * fs.f_bfree;
+            point.size = (static_cast<uint64_t>(fs.f_bsize)
```

Comment:
Recapping our hallway convo:
* the parens are not an annoyance, but it is more cognitive load, because I assume parens are there for semantic purposes (I'm not familiar with the idiom of using parens as a trigger to the editor for indenting)
* by default, projects using leatherman to define cpplint filters will skip the line length check: https://github.com/puppetlabs/leatherman/blob/e5d7b1767bb7352ce13fa684ea3edfd4304684ea/cmake/leatherman.cmake#L135
* facter has quite a few lines > 80 chars:
```
facter git:(master) ag --cpp '.{81}' . | wc -l
    1782
```

It is clearly a style/idiom thing, so just take the above comments as background for why I asked the question in the first place.